config.sample.json

```
@@ -2,17 +2,17 @@
     "default_server_config": {
         "m.homeserver": {
             "base_url": "https://matrix.i.tchap.gouv.fr",
-            "server_name": "Matrix Internes Preprod"
+            "server_name": "Internes - Preprod"
         },
         "m.identity_server": {
             "base_url": "https://matrix.i.tchap.gouv.fr"
         }
     },
-    "disable_custom_urls": false,
-    "disable_guests": false,
+    "disable_custom_urls": true,
```

```suggestion
    "disable_custom_urls": false,
```
On a besoin de changer de homeserver parce qu'il y en a plusieurs

---

Yes, c'était vraiment configuré pour un PoC, avec volonté de changer plus tard